drizzle-orm/src/primary-key.ts

```diff
@@ -9,3 +9,9 @@ export abstract class PrimaryKey {
 
 	constructor(readonly table: Table, readonly columns: AnyColumn[]) {}
```

Comment:
I noticed that the other PrimaryKey classes don't extend this class.  I'm not sure if thats expected, or if this is left over from previous refactoring.  I didn't want to over complicate this PR, so I left this as is